crates/topos-tce-broadcast/benches/task_manager.rs

```diff
@@ -82,7 +82,9 @@ pub async fn processing_double_echo(n: u64, validator_store: Arc<ValidatorStore>
         .collect::<Vec<_>>();
 
     for cert in &certificates {
-        double_echo.broadcast(cert.certificate.clone(), true).await;
+        _ = validator_store
+            .insert_pending_certificate(&cert.certificate)
+            .unwrap();
```

Comment:
Let's add error handling here. What does it mean if we cannot add it to the store? What should we do then?

Reply:
as it is for `cargo benchmark` that we do not currently use I was thinking it was ok to panic with the `StorageError` directly instead of creating a complex error handling.